README.md

```diff
@@ -0,0 +1,7 @@
+# rmw_implementation
+
+The decision which ROS middleware implementation should be used for C++.
```

Comment:
I think this method does not take the decision, maybe a better wording would be using mapping/loadding
```suggestion
Implementation of ROS middleware functions used in C++.
```